boards/pba-d-01-kw2x/include/periph_conf.h

```diff
@@ -47,6 +47,7 @@ extern "C"
 #define KINETIS_MCG_PLL_FREQ              48000000
 
 #define CLOCK_CORECLOCK                   KINETIS_MCG_PLL_FREQ
+#define CLOCK_BUSCLOCK                    (48e6)
```

Comment:
I just copied it from the defines below, did not even start to wonder what this does :-)